v2/kafka-to-gcs/README.md

```diff
@@ -0,0 +1,98 @@
+# Kafka To Google Cloud Storage Dataflow Template
+
+The [KafkaToGCS](src/main/java/com/google/cloud/teleport/v2/templates/KafkaToGCS.java) pipeline reads message from Kafka topic(s) and stores to Google Cloud Storage bucket in user specified file format. The sink data can be stored in a Text, Avro or a Parquet File Format.

```

Comment:
I believe it is new-line-delimited JSON for Text. Can you confirm?

Reply:
No, it is not new-line-delimited JSON. As of now we are considering only the Value Part of the Message Payload